pkg/ebpf/c/bpf_bypass.h

```diff
@@ -0,0 +1,51 @@
+#ifndef __BPF_BYPASS_H__
+#define __BPF_BYPASS_H__
+
+#include "compiler.h"
+#include "map-defs.h"
+
+// default to size 1 so it doesn't accidentally break programs that aren't using it
+BPF_ARRAY_MAP(program_bypassed, u32, 1)
```

Comment:
Worth it to make this per cpu when available?

Reply:
It does, but that is handled in the manager.